experiments/exp163_bert.py

```diff
@@ -0,0 +1,316 @@
+"""
+Experiment 163: Compare BERT vs Fasttext classifiers (in this case, with MMLU positive examples).
```

Comment:
Say a bit more - what are the positive/negative examples, what are we selecting from, what are we training, for how long etc.  Doesn't have to be long, but just 2 sentences.